anthos-attached-clusters/modules/attached-install-manifest/variables.tf

```diff
@@ -40,3 +40,9 @@ variable "attached_cluster_name" {
   description = "Name for the attached cluster resource"
   type        = string
 }
+
+variable "helm_timeout" {
+  description = "(Optional) Time in seconds to wait for Helm operations to complete."
+  type        = number
+  default     = null
```

Comment:
qq: What's the timeout when it's null?

Reply:
Sure - When not specific / `null`, it will use the provider default, which in v2.17.0 is `300s`.